### PR TITLE
Merge dependabot cron

### PR DESCRIPTION
**Pull request by iwahbe**
Call `./scripts/merge_dependabot.sh` every 4 days.

### CODE REVIEW

File: .github/workflows/merge-dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: merge-dependabot
+
+on:
+  workflow_dispatch:
```
Comment (thomas11) [inline .github/workflows/merge-dependabot.yml:4]: We use `workflow_dispatch: {}` everywhere else that I can see from a quick grep (unless there are actual arguments). Is the `{}` not necessary?

Comment (iwahbe) [reply .github/workflows/merge-dependabot.yml]: I'm not 100% sure. I added the `{}` to be like other workflows.

File: .github/workflows/merge-dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: merge-dependabot
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    # 4:00AM every 4 days.
+    - cron:  '0 4 01/4 * *'
```
Comment (guineveresaenger) [inline .github/workflows/merge-dependabot.yml:7]: TIL you can't say "every fourth day" in cron. 
Another option would be to run every Monday and Thursday, which is slightly more frequent but less crowded around the first of the month? Either way is fine though.

**Review by guineveresaenger — APPROVED** [review]
LGTM

**Review by thomas11 — APPROVED** [review]

File: .github/workflows/merge-dependabot.yml (hunk 2 above)
Comment (iwahbe) [reply .github/workflows/merge-dependabot.yml:7]: > TIL you can't say "every fourth day" in cron.

Maybe you can, but I don't know how.

**Pull request merged**